community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/util.py

```diff
@@ -1824,6 +1844,8 @@ def get_reservation_details(self, project:str, zone:str, name:str, bulk_insert_n
             policies=policies,
             deployment_type=reservation.get("deploymentType"),
             reservation_mode=reservation.get("reservationMode"),
+            assured_count=reservation.get("assuredCount"),
+            delete_at_time=parse_gcp_timestamp(reservation.get("deleteAtTime")),
```

Comment:
If this can truly be `None` we should do some type checking before sending it into this function